### ISSUE TITLE
Automatically default fixtures option to __dirname + /fixtures

### ISSUE THREAD

**Issue by Flet** [Flet/tape-nock#2]
This could be a bit tricky, but it would be nice to just default it to /fixtures instead of having to include it every time.

**Comment by goodmind** [comment]
Something like this?

```
nockBack.fixtures = path.join(__dirname, 'fixtures') || nockOpts.fixtures
```

**Comment by Flet** [comment]
As I understand, `__dirname` is always relative to the currently executing script, so using `__dirname` inside of tape-nock will not work...

Using something like `callsite` could work: https://www.npmjs.com/package/callsite

Maybe something like

``` js
var callsite = require('callsite')
var path = require('path')

var stack = callsite()
var callerFile = stack[1].getFileName()

var fixuresDir = path.join(path.dirname(callerFile), 'fixtures')
console.log('fixtures directory:', fixturesDir) 
```

**Comment by goodmind** [comment]
Hm, I think this not worth it

**Comment by goodmind** [comment]
@Flet what you think?

**Comment by goodmind** [comment]
@Flet I think it would be better to use 
```js
path.dirname(module.parent.filename)
```

**Comment by goodmind** [comment]
@Flet ?

**Comment by goodmind** [comment]
Hey @Flet

**Comment by Flet** [comment]
Hi

**Comment by goodmind** [comment]
What about 
```js
module.parent.filename
```

**Comment by big-kahuna-burger** [comment]
#16

**Comment by Flet** [comment]
Closed via #16
